Source/Core/Core/NetPlayServer.h

```diff
@@ -71,6 +75,11 @@ class NetPlayServer : public TraversalClientClient
     u32 ping;
     u32 current_game;
 
+#ifdef _WIN32
+    HANDLE qos_handle;
```

Comment:
These can just be a `void*` and `unsigned long` to prevent dragging OS-specific headers into the Core headers where it doesn't need to be). Which allows placing it in the cpp file, preventing it from leaking out into everything else that includes this header.